project/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
```

Comment:
If this text will be presented to users then it will be better to write "Place our project id here".

Reply:
I fixed this to be called capstone project!